piker/calc.py

```diff
@@ -20,10 +20,17 @@
 import math
 import itertools
 
+from bidict import bidict
+
+
+_mag2suffix = bidict({3: 'k', 6: 'M', 9: 'B'})
```

Comment:
are there any other ones i'm not thinking of?

`m` seems like a worse version of `k`.